website/docs/r/elasticsearch_domain.html.markdown

```diff
@@ -240,6 +241,11 @@ The following arguments are supported:
 * `enabled` - (Required) Whether to enable encryption at rest. If the `encrypt_at_rest` block is not provided then this defaults to `false`.
 * `kms_key_id` - (Optional) The KMS key id to encrypt the Elasticsearch domain with. If not specified then it defaults to using the `aws/es` service KMS key.
 
+**domain_endpoint_options** supports the following attributes:
+
+* `require_https` - (Required) wether or not to require HTTPS
+* `tls_security_policy` - (Optional) the TLS security policy that needs to be applied to the HTTPS endpoint. Defaults to `Policy-Min-TLS-1-2-2019-07`
```

Comment:
We would prefer not to choose a default in this situation, but should still mention the valid values.

```suggestion
* `tls_security_policy` - (Optional) The name of the TLS security policy that needs to be applied to the HTTPS endpoint. Valid values:  `Policy-Min-TLS-1-0-2019-07` and `Policy-Min-TLS-1-2-2019-07`. Terraform will only perform drift detection if a configuration value is provided.
```